### PR TITLE
[docs] Improve README.md to include usage instructions for platform.sh projects

### PR DESCRIPTION
**Pull request by gilzow**
Per slack conversation, updates README to include usage instructions for platform.sh projects

### CODE REVIEW

**Comment by rfay** [comment]
Thank you!

File: README.md [hunk 1]
```diff
@@ -4,20 +4,28 @@
...
 
-## Installing
+## Using with a Platform.sh project
+1. Make sure you have [DDEV v1.21.1+ installed](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/)
+2. Clone your project repository (e.g. `platform get <projectid>`)
+3. `cd` into your project directory
+4. Run `ddev config` and answer the questions as appropriate
+5. Run `ddev get platformsh/ddev-platformsh` and answer the questions as appropriate
+6. Run `ddev start`
+7. Run `ddev pull platform` to retrieve a copy of the database and contents from the project's file mounts from the environment you entered in step #5
```
Comment (lolautruche) [inline README.md:14]:
```suggestion
7. (Optional) Run `ddev pull platform` to retrieve a copy of the database and contents from the project's file mounts from the environment you entered in step #5
```

File: README.md [hunk 2]
```diff
@@ -4,20 +4,28 @@
...
 ## What does it do right now?
 
 * Takes your checked-out Platform.sh project and configures DDEV based on that information.
-  * PHP and Database version
-  * hooks are converted to DDEV post-start hooks
-  * A working `ddev pull platform` integration with all mounts is created.
-  
+    * PHP and Database version
+    * hooks are converted to DDEV post-start hooks
+    * A working `ddev pull platform` integration with all mounts is created.
```
Comment (lolautruche) [inline README.md:27]: We should mention that we support configured services, and which ones are supported.

**Review by lolautruche — CHANGES_REQUESTED** [review]

**Review by lolautruche — APPROVED** [review]

**Comment by lolautruche** [comment]
Should be fine now.
Can we merge @rfay ?

**Review by rfay — APPROVED** [review]

**Comment by rfay** [comment]
Thanks!

**Pull request merged**